src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -163,4 +168,22 @@ public static void showPassengerAtIndex(Model model, Index targetIndex) {
         assertEquals(1, model.getFilteredPassengerList().size());
     }
 
+    /**
+     * Updates {@code model}'s filtered list to show only the pool at the given {@code targetIndex} in the
+     * {@code model}'s address book.
+     */
+    public static void showPoolAtIndex(Model model, Index targetIndex) {
+        assertTrue(targetIndex.getZeroBased() < model.getFilteredPoolList().size());
+
+        Pool pool = model.getFilteredPoolList().get(targetIndex.getZeroBased());
+
+        final String keywords = pool.getDriver().getName().fullName + " "
+                + pool.getTripDayAsStr() + " "
+                + pool.getTripTimeAsStr();
+        final String[] splitKeywords = keywords.split("\\s+");
+        model.updateFilteredPoolList(new DriverNameContainsKeywordsPredicate(Arrays.asList(splitKeywords[0])));
+
+        assertEquals(1, model.getFilteredPoolList().size());
```

Comment:
Is there a better way to test if the right pool is shown instead of checking the size of the filteredlist? maybe getting the pool object from the filtered list and asserting that it equals an expected pool?